### DISCUSSION TITLE
[Feature request] Hide family label

### DISCUSSION THREAD

**Discussion by Manu99it** [ChristopherKlay/StadiaEnhanced#104] [Ideas & Requests]
Would be awesome an option to hide the "Shared by family member" label how already you can hide the PRO label, possibly separated. You could also use the same button with "default", "hide pro label", "hide family label", "hide both"

**Comment by ChristopherKlay** [comment]
Will be included with the next version.

I changed "Pro Labels" to "Game Labels" for it and fixed a couple of bugs for the next release (:

**Reply by Manu99it** [reply]
Awesome, thank you!
